src/Dfc.CourseDirectory.Core/DataStore/CosmosDb/Queries/UpdateProviderOnboarded.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using OneOf;
+using OneOf.Types;
+
+namespace Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries
+{
+    public class UpdateProviderOnboarded : ICosmosDbQuery<OneOf<NotFound, Success>>
+    {
+        public Guid ProviderId { get; set; }
+
+        public string UpdatedBy { get; set; }
```

Comment:
```suggestion
        public UserInfo UpdatedBy { get; set; }
```

Reply:
Doing so will change the behaviour as we currently pass through a magic string - was your intention here to pass the full `UserInfo` object or future proof?

Reply:
It's for consistency and future-proofing. Every other handler takes a `UserInfo` property on the command for setting the `{Created|Updated}By` field.

Reply:
Implementing that properly will change the behaviour. Do we want to do this?

Reply:
For me yes. `UpdatedBy` should be the email of the user making the change.